test/others/skip-file-rwx-check/run.sh

```diff
@@ -8,21 +8,22 @@ set -o xtrace
 source ../env.sh
 
 make clean
+make tester
 touch testfile
 chmod +w testfile
-tail --follow testfile &
-tailpid=$!
-if ! "$criu" dump --tree=$tailpid --shell-job --verbosity=4 --log-file=dump.log
+./tester testfile &
```

Comment:
nit: you can use bash to open a file descriptor:
bash -c 'exec 3<testfile; while :; do sleep 1; done' &

Reply:
Thanks. Using that now and dropping the C file.